x-pack/plugin/ent-search/src/test/java/org/elasticsearch/xpack/application/search/action/RestPutSearchApplicationActionTests.java

```diff
@@ -28,6 +28,15 @@ public void testWithNonCompliantLicense() throws Exception {
         );
     }
 
+    public void testInvalidRequestWithNonCompliantLicense() throws Exception {
+        checkLicenseForRequest(
+            new FakeRestRequest.Builder(NamedXContentRegistry.EMPTY).withMethod(RestRequest.Method.PUT)
+                .withParams(Map.of("name", "my-app"))
+                .withContent(new BytesArray("{}"), XContentType.JSON)
```

Comment:
we can even put like a clearly invalid parameter `.withContent(new BytesArray("{\"invalid_parameter\": \"invalid_value\"}"), XContentType.JSON)`

Reply:
Nice suggestion, I'll change that!